### PR TITLE
patch RU translation

### PR DESCRIPTION
**Pull request by tonymorony**
not translated yet labels mentioned in https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1008 since it's sounds not clear

task list:

- [ ] check if app look good in russian (no overflow)

### CODE REVIEW

**Comment by smk762** [comment]
Some overflow here

image

Rest looks ok.

**Comment by Milerius** [comment]
> Some overflow here
> 
> image
> 
> Rest looks ok.

@tonymorony there is no shorter wording for this traduction ? If the answer is no please @ssakone make this modal/button dynamic based on text size

**Comment by tonymorony** [comment]
`@tonymorony there is no shorter wording for this traduction ? If the answer is no please @ssakone make this modal/button dynamic based on text size`

I'll think about it a bit later and commit shorter version

**Comment by smk762** [comment]
Just noticed there is also overflow once a coin type is selected in addressbook - 
image

Though this was mentioned in the PBC feedback PDF also, and is an issue on other languages too
image
image
image

**Comment by smk762** [comment]
Similarly Turkish is affected by the coin activation menu overflow also
image

French and English ok.

**Comment by tonymorony** [comment]
@SylEze @Milerius could you please advise on the English file update? (https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1015#issuecomment-867492295) 

there are new labels in code but not in translation file:
image

**Comment by Milerius** [comment]
> @SylEze @Milerius could you please advise on the English file update? ([#1015 (comment)](https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1015#issuecomment-867492295))
> 
> there are new labels in code but not in translation file:
> image

You need to sync with last dev to have last labels
